mesonbuild/minstall.py

```diff
@@ -418,8 +419,9 @@ def do_copyfile(self, from_file: str, to_file: str,
             else:
                 # Remove this entire branch when changing the behaviour to duplicate
                 # symlinks rather than copying what they point to.
-                print(symlink_warning)
-                self.copy2(from_file, to_file)
+                if follow_symlinks is None:
+                    print(symlink_warning)
+                self.copy2(from_file, to_file, follow_symlinks=follow_symlinks)
```

Comment:
In practice I think the reason this has always been an install-time warning was because we don't really know it's a symlink before trying to install it.

Reply:
Fair enough, this should still do `follow_symlinks = False` in the `is None` block

Reply:
> Fair enough, this should still do `follow_symlinks = False` in the `is None` block

on it